src/resource/factory/SoH/ResourceLoader.cpp

```diff
@@ -102,11 +100,11 @@ namespace Ship
     }
 
     if (result != nullptr) {
-        result->file = FileToLoad;
-        result->resType = resourceType;
+        result->File = FileToLoad;
+        result->ResType = resourceType;
```

Comment:
`ResourceType` instead maybe?

Reply:
This would probably be ambiguous with the type `Ship::ResourceType` and thus fail in gcc.